### DISCUSSION TITLE
Inheritance for environments

### DISCUSSION THREAD

**Discussion by zandivx** [lvjr/tabularray#35] [Ideas]
When defining new tabularray environments with \NewTblrEnviron I often find myself copying and pasting existing definitions of other environments I've created earlier. I think it would be a great option to be able to refer to existing environments and inherit their options (something like \UseTblrTemplate but for environments).

**Comment by lvjr** [comment]
Good idea.  This feature may be added later this year.

**Reply by lvjr** [reply]
Created issue #37.
